lib/runner/Configuration.js

```diff
@@ -17,7 +17,7 @@ module.exports = class Configuration {
         return Configuration.singleton;
     }
 
-    async load(json, path) {
+    async load(json, pathName) {
```

Comment:
Can you add a comment on this - what does the pathname represent and what is it used for?